main.nf

```diff
@@ -18,6 +18,8 @@
 include { CELLPAINTING  } from './workflows/cellpainting'
 include { PIPELINE_INITIALISATION } from './subworkflows/local/utils_nfcore_cellpainting_pipeline'
 include { PIPELINE_COMPLETION     } from './subworkflows/local/utils_nfcore_cellpainting_pipeline'
+
+include { CYTOTABLE } from './modules/nf-core/cytotable'
```

Comment:
```suggestion
include { CYTOTABLE } from './modules/local/cytotable'
```
All of the files need to be moved to this directory as well. We are developing this modules as a module that is local to this pipeline instead of a module within the nf-core/modules library.